js/index.js

```diff
@@ -1,9 +1,11 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
-
-  //... your code goes here
+  let price = document.querySelector('.price span').innerHTML();
```

Comment:
Ok, I see where you are trying to go with this. Keep in mind, however, that `innerHTML` is an attribute in the DOM element and not a function. Calling `innerHTML()` results in a TypeError to be thrown in upon running this. 

To get your updateSubtotal working, consider having the following steps in your logic:

1. Get DOM elements that hold price and quantity. 
2. Extract values from DOM elements.
3. Calculate total values.
4. Get DOM element that holds the subtotal value for the product.
5. Set the product subtotal to the corresponding DOM element.
6. Return subtotal value so it can be used later.